spec/models/user_spec.rb

```diff
@@ -0,0 +1,98 @@
+require "rails_helper"
+require 'faker'
```

Comment:
Use FFaker and configure it in rails/spec helper if needed

https://github.com/ffaker/ffaker

Reply:
done